subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -213,6 +213,34 @@ The created task is named “`publish«_PUBNAME_»PublicationToMavenLocal`”.
 
 The resulting task in this example is named “`publishMavenJavaPublicationToMavenLocal`”. This task is wired into the `publishToMavenLocal` lifecycle task. Executing “`gradle publishToMavenLocal`” builds the POM file and all of the artifacts to be published, and “installs” them into the local Maven repository.
```

Comment:
I think it's in quotes because "install" is Maven's terminology for copying the artifacts to the local Maven repository. So, I'm wondering whether we might actually replace it with "copies" (without the quotes, of course 😉). Thoughts?